src/XMLimport.cpp

```diff
@@ -825,6 +825,9 @@ void XMLimport::readHostPackage(Host* pHost)
     pHost->mMapperUseAntiAlias = attributes().value(QStringLiteral("mMapperUseAntiAlias")) == YES;
     pHost->mMapperShowRoomBorders = readDefaultTrueBool(QStringLiteral("mMapperShowRoomBorders"));
     pHost->mEditorAutoComplete = (attributes().value(QStringLiteral("mEditorAutoComplete")) == YES);
+    if (!attributes().hasAttribute("mEditorShowBidi") || (attributes().value(QStringLiteral("mEditorShowBidi")) == YES)) {
```

Comment:
I've mentioned this before but given that an XML file is supposed to be readable by both Humans AND Machines it does make sense to make the attributes/elements have identifiers that are understandable by the former group - obviously it is hard to change existing ones retrospectively but for new ones it makes sense (to me anyway) to give clear names rather than, possibly cryptic, variable names for these thing:
```suggestion
    if (!attributes().hasAttribute("ShowBiDiMarksInEditor") || (attributes().value(QStringLiteral("mEditorShowBidi")) == YES)) {
```

P.S. Since BiDi is a contraction of "Bi-Directional" I think it is not inappropriate to capitalise the "D" as well as the "B"...

Reply:
Obviously the same sort of change will be needed in the `./src/XMLExport.cpp` file!

Reply:
I don't agree with this because it's not consistent with how other keys are named and it's not much more of an improvement in readability... 

Plus we are not expecting people to be reading these XML files all that much, in ten years people have not been spending a lot of time inspecting the Host part of it.

Reply:
Oh, well.

🤷‍♂️